db/migrations/003_repositories_add_quality.rb

```diff
@@ -0,0 +1,9 @@
+Sequel.migration do
+  change do
+  	alter_table(:repositories) do
+  	  add_column :flog_score, String
+  	  add_column :flay_score, String
+      add_column :rubocop_errors, String
```

Comment:
it seems that the indentation is weird here. make sure you use space and not tab